LoopKitUI/Views/QuantityPicker.swift

```diff
@@ -127,7 +140,21 @@ public struct QuantityPicker: View {
         }
     }
 
-    private var unitLabelSpacing: CGFloat { 8 }
+    private var unitLabelSpacing: CGFloat {
+        if useSizeablePicker {
+            return 0
+        } else {
+            return 8
+        }
+    }
+    
+    private var useSizeablePicker: Bool {
```

Comment:
Any reason to not just use the custom picker on any OS version? Seems like it would eliminate a lot of code here.

Reply:
I'm fine with that...that was what [my question above](https://github.com/tidepool-org/LoopKit/pull/425#discussion_r765096638) was about.  The only reason I can think of is I'm not really thrilled with this workaround, and it would be great to be able to revert it easily.  If eliminating extra code is higher priority, I'll just put a big comment in instead.